packages/shared/src/lib/image.ts

```diff
@@ -10,4 +10,9 @@ export const cloudinary = {
     supercharge:
       'https://daily-now-res.cloudinary.com/image/upload/v1665563545/public/supercharge.png',
   },
+  notifications: {
+    browser:
+      'https://daily-now-res.cloudinary.com/image/upload/v1670221485/public/notification.png',
+    bell: '',
```

Comment:
Please sort this one out before merging :)

Reply:
This is not used here. Just the browser. The bell one is found on the other PR.